topology-graph.js

```diff
@@ -28,7 +28,7 @@
     /* A cache to prevent jumping when rapidly toggling views */
     var cache = { };
 
-    function topology_graph(selector, force, notify) {
+    function topology_graph(selector, force, notify, customRadius) {
```

Comment:
This isn't public API ... yet ... until we have/need a "jQuery" implementation. Only the angular code below calls it for now.